_i18n/de/resources/moneropedia/wallet.md

```diff
@@ -1,18 +1,17 @@
 ---
-terms: ["wallet", "wallets"]
-summary: "A wallet stores the information necessary to send and receive Monero"
+terms: ["wallet", "wallets", "Wallet", "Wallets"]
+summary: "Ein Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A Monero account, or wallet, stores the information necessary to send and receive Moneroj.  In addition to sending and receiving, the Monero Wallet software keeps a private history of your transactions and allows you to cryptographically sign messages.  It also includes Monero mining software and an address book.
+Ein Monero-Konto beziehungsweise ein Wallet speichert die Informationen, die zum Senden und Empfangen von Monero benötigt werden. Zusätzlich zum Senden und Empfangen führt und bewahrt die Wallet-Software von Monero eine private Chronik deiner Transaktionen und ermöglicht es dir zudem, Nachrichten kryptografisch zu signieren. Sie enthält außerdem ein Programm zum Minen von Monero und ein Adressbuch.
 
-The term "hot wallet" describes a Monero @account which is connected to the Internet.  You can send funds easily but security is much lower than a cold wallet.  Never store large amounts of cryptocurrency in a hot wallet!
+Ein Spar-Wallet ("Hot-Wallet") ist ein mit dem Internet verbundenes Monero-@Konto. Du kannst Gelder zwar bequem versenden, die Sicherheit ist allerdings sehr viel niedriger als es bei einem Spar-Wallet der Fall ist. Bewahre niemals große Mengen von Kryptowährung in einem Zahlungs-Wallet auf! 
```

Comment:
I'm still confused by "wallet" and "account".. I think I've seen it getting mixed up or used interchangeably in other places too..

```suggestion
Ein Zahlungs-Wallet ("Hot-Wallet") ist ein mit dem Internet verbundenes Monero-@Konto. Du kannst Gelder zwar bequem versenden, die Sicherheit ist allerdings sehr viel niedriger, als es bei einem Spar-Wallet der Fall ist. Bewahre niemals große Mengen von Kryptowährung in einem Zahlungs-Wallet auf! 
```

Reply:
Yes. We don't use them consistently and there is disagreement about what each term should be used. As you have seen, the documentation needs a lot of work :)